training/src/app/modules/chuck-norris/components/joke-generator/joke-generator.component.html

```diff
@@ -0,0 +1,34 @@
+<div [ngClass]="{'search-style': isSearchMode}" *ngIf="isSearchMode" class="buttons-container">
+    <a>Key word(s)</a>
+    <app-search (generateJokesOnSearchMode)="onSearch($event)"
+                (generateJokesOnClickMode)="this.searchKey = $event"
+    ></app-search>
+    <button [ngClass]="{'disabled-style': this.searchKey.length < 3}"
+            [disabled]="this.searchKey.length < 3"
+            (click)="onClick(this.searchKey)" class="btn-three" id="btn-three" mat-flat-button>Go!
+    </button>
+</div>
+
+<div *ngIf="!isSearchMode" class="buttons-container">
```

Comment:
instad of doing:  <div *ngIf=true>. and <div *ngIf=false>.
you can do if else, please google it